src/senaite/core/browser/controlpanel/manufacturers/view.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE.
+#
+# SENAITE.CORE is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2024 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+import collections
+
+from bika.lims import api
+from bika.lims import bikaMessageFactory as _
+from bika.lims.utils import get_link_for
+from senaite.app.listing import ListingView
+from senaite.core.catalog import SETUP_CATALOG
+
+
+class ManufacturersView(ListingView):
+    """Displays all available manufacturers
+    """
+
+    def __init__(self, context, request):
+        super(ManufacturersView, self).__init__(context, request)
+
+        self.catalog = SETUP_CATALOG
+
+        self.contentFilter = {
+            "portal_type": "Manufacturer",
+            "sort_on": "sortable_title",
```

Comment:
Please filter the results by path too:

```python
"path": {
    "query": api.get_path(self.context),
    "depth": 1,
}
```
 This will make the listing to display correct results even if other products allow the creation of objects from `Manufacturer` type outside of `setup.manufacturers` folder